ryan_heathers/test/greetingSpec.js

```diff
@@ -0,0 +1,8 @@
+var expect = require('chai').expect;
+var greet = require('../lib/greeting').greet;
+
+describe('Greeting function', function() {
+  it('should return string of "hello" plus name', function() {
+    expect(greet('Albert')).to.equal('hello Albert');
+  });
+});
```

Comment:
Always leave a trailing line at the end of your js files.
Do you have a test for your command line utility?
Nice styling of your code!

Come see me and I'll show you how to untrack your node_modules